### PR TITLE
Update co-reference_resolution.md

### PR DESCRIPTION
**Pull request by LowinLi**
新增 共指消解 评测方法 两个

### CODE REVIEW

**Comment by Huang17** [comment]
Hi Lowin, 感谢你帮助我们完善ChineseNLP项目。关于你提到的`BLANC`和 `the link based entity aware LEA metric`这两个评测指标，有用到这两个指标的相关论文么？

**Comment by LowinLi** [comment]
Hi 不客气！

+ BLANC
[2011论文](https://www.cs.cmu.edu/~hovy/papers/10BLANC-coref-metric.pdf)
[2014论文](https://www.aclweb.org/anthology/P14-2005.pdf)
+ the link based entity aware LEA metric
[2016论文](https://www.aclweb.org/anthology/P16-1060.pdf)

**Comment by Huang17** [comment]
Looks great!

File: docs/co-reference_resolution.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Average of F1-scores returned by these three precision/recall metrics:
 - MUC.  
 - B-cubed.  
 - Entity-based CEAF.  
+- BLANC.
+- the link based entity aware LEA metric.
```
Comment (Huang17) [inline docs/co-reference_resolution.md:27]: One minor suggestion, could you modify line 27 into `- Link-Based Entity-Aware metric (LEA).` ? This is a more official name based on the original [paper](https://www.aclweb.org/anthology/P16-1060.pdf).

Comment (LowinLi) [reply docs/co-reference_resolution.md:27]: of course:)

**Comment by Huang17** [comment]
great! PR merged.

**Pull request merged**